console/program/src/data_types/plaintext_type/mod.rs

```diff
@@ -26,14 +26,60 @@ pub enum PlaintextType<N: Network> {
     /// A literal type contains its type name.
     /// The format of the type is `<type_name>`.
     Literal(LiteralType),
-    /// An struct type contains its identifier.
+    /// A struct type contains its identifier.
     /// The format of the type is `<identifier>`.
     Struct(Identifier<N>),
+    /// An external struct type contains its locator.
```

Comment:
Nit. Can we change the ordering so that `ExternalStruct` comes that the end? And ditto for all of the enums.

This is to ensure that the serialization variants follow the enum declaration order.